qrisk/tests/test_stats.py

```diff
@@ -79,49 +137,140 @@ def test_annual_volatility(self, returns, period, expected):
         )
 
     @parameterized.expand([
-        (simple_rets, 1.2333396776895436),
+        (s_2.pct_change().dropna(), -2.3992211554712197)
```

Comment:
can we craft an input that gives a more intuitive almar ratio? I have no intuition for how accurate this is.